### DISCUSSION TITLE
ScriptReference/Build.Reporting.BuildFile-role

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#629] [Notes]
# ScriptReference/Build.Reporting.BuildFile-role

https://docs.unity3d.com/ScriptReference/Build.Reporting.BuildFile-role.html

**Comment by qwe321qwe321qwe321** [comment]
This property doesn't follow the CommonRoles class in Unity 2021 and later versions.
[Unity Forum discussion](https://forum.unity.com/threads/buildfile-role-worked-inconsistent-with-it-used-to-be-after-unity-2021.1554896/)
